sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -55,14 +55,9 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 04799e735a7ddd0839568b25bf03fa8f77160b7d
-  # Merge: 534071e5 4cf02cb9
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Mon Nov 20 10:24:50 2023 +0000
-  #     Merge pull request #1048 from callumfare/opencl_fix_urMemBufferCreate_leak
-  #     [OpenCL] Fix memory leak and coverity issue with struct-to-array casts
-  set(UNIFIED_RUNTIME_TAG 04799e735a7ddd0839568b25bf03fa8f77160b7d)
+  # DO NOT MERGE
+  set(UNIFIED_RUNTIME_REPO https://github.com/hdelan/unified-runtime.git)
+  set(UNIFIED_RUNTIME_TAG revert-hip-prefetch)
```

Comment:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 28cf40fa19e7c1fed139d219abe563f102a43853
  # Merge: 109ed46e 841a2870
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Fri Nov 24 12:12:09 2023 +0000
  #     Merge pull request #936 from hdelan/revert-hip-prefetch
  #     [HIP] Revert add prefetch for USM hip allocations a6b8fa66b537753415d24076f…
  set(UNIFIED_RUNTIME_TAG v0.8.0)
```

Reply:
@hdelan I went ahead and applied this patch and also merged in the latest sycl branch changes.